modules/gitops-installing-argocd-cli-on-linux.adoc

```diff
@@ -0,0 +1,76 @@
+// Module is included in the following assemblies:
+//
+// * installing_gitops/installing-argocd-gitops-cli.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-installing-argocd-cli-on-linux"]
+= Installing the {gitops-title} CLI on Linux
+
+For Linux distributions, you can download the {gitops-shortname} `argocd` CLI as a `tar.gz` archive.
+
+.Procedure
+
+. Download the relevant CLI tool.
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/argocd-cli/1.12.0/argocd-cli-darwin-amd64.tar.gz[Linux (x86_64, amd64)]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/argocd-cli/1.12.0/argocd-cli-darwin-s390x.tar.gz[Linux on {ibmzProductName} and {linuxoneProductName} (s390x)]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/argocd-cli/1.12.0/argocd-cli-darwin-ppc64le.tar.gz[Linux on {ibmpowerProductName} (ppc64le)]
+
+* link:https://mirror.openshift.com/pub/openshift-v4/clients/argocd-cli/1.12.0/argocd-cli-darwin-arm64.tar.gz[Linux on ARM (aarch64, arm64)]
+
++
+You can also download any version of the {gitops-shortname} `argocd` CLI by navigating to that version’s corresponding directory in the link:https://mirror.openshift.com/pub/openshift-v4/clients/argocd-cli/[`argocd` CLI client download mirror].
```

Comment:
It's a _bit_ anthropomorphizing to say that a version has a directory. Slight rewording:

"You can also download any version of the {gitops-shortname} `argocd` CLI by navigating to the corresponding directory for the desired version"